src/main/java/frc/robot/constants/Constants.java

```diff
@@ -72,15 +82,15 @@ interface Vision {
         //L = Left, R = Right, F = Forward, B = Backward (Facing)
         Transform3d ROBOT_TO_FL_APRILTAG_CAM = new Transform3d( // X Z Y
                 new Translation3d(Units.inchesToMeters(11.862), Units.inchesToMeters(12.681), Units.inchesToMeters(8.947)),
-                new Rotation3d(Units.degreesToRadians(172.57530061265695), Units.degreesToRadians(-25), Units.degreesToRadians(25))
+                new Rotation3d(Units.degreesToRadians(-7.44802852475369), Units.degreesToRadians(-25), Units.degreesToRadians(25))
         );
 
         /**
          * Standard deviations of the supplied pose estimate (before vision, likely to be solely wheel odometry)
          */
-        Vector<N3> STATE_STD_DEVS = VecBuilder.fill(0.1, 0.1, Units.degreesToRadians(5));
-        Vector<N3> VISION_MEASUREMENT_STD_DEVS = VecBuilder.fill(0.9, 0.9, Units.degreesToRadians(80));
-        double MULTI_TAG_MAX_AMBIGUITY = 0.3;
+        Vector<N3> STATE_STD_DEVS = VecBuilder.fill(0.1, 0.1, Units.degreesToRadians(1));
+        Vector<N3> VISION_MEASUREMENT_STD_DEVS = VecBuilder.fill(0.6, 0.6, Units.degreesToRadians(30));
```

Comment:
The std. devs. here can stay for now. Will make adjustments + more changes soon.